Makefile

```diff
@@ -49,7 +49,7 @@ test-integration-without-spiceai-dataset:
 
 .PHONY: test-bench
 test-bench:
-	@cargo bench -p runtime
+	@cargo bench -p runtime --features postgres
```

Comment:
Propose to be consistent with spark and enable bench for non-default connectors in workflow action (postgres seems already enabled): https://github.com/spiceai/spiceai/blob/trunk/.github/workflows/benchmarks.yml#L26